puente/ext.py

```diff
@@ -5,10 +5,10 @@
 
 
 @jinja2.contextfunction
-def _gettext_alias(context, text, *args, **kwargs):
+def _gettext_alias(context, *args, **kwargs):
```

Comment:
So, this mimics what Jinja2 does here:

https://github.com/mitsuhiko/jinja2/blob/master/jinja2/ext.py#L130